user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,103 @@
+
+Throughout this document we'll be using the following terms to refer to the two components of your Travis CI Enterprise installation:
+
+- `Platform machine`: The virtual machine that runs most of the Travis web components. This is the machine your domain is pointing to.
+- `Worker machine`: The worker machine(s) run your builds.
+
+## Backups
+
+This section explains how you integrate Travis CI Enterprise in your backup strategy. Here, we'll talk about two topics:
+
+- The encryption key
+- The data directories
+
+### Encryption key
+
+Without the encryption key you cannot access the information in your production database. To make sure that you can always recover your database, make a backup of this key:
+
+1. open a ssh connection to the platform machine
+2. run `travis bash`. This will open a bash session with `root` privileges into the Travis container.
+3. Then run `cat /usr/local/travis/etc/travis/config/travis.yml | grep -A1 encryption:`. Create a backup of the value returned by that command by either writing it down on a piece of paper or storing it on a different computer.
+
+> Without this key the information in the database is not recoverable.
+
+### Create a backup of the data directories
+
+The data directories are located on the platform machine and get mounted into the Travis container. In these directories you'll find files from RabbitMQ, Postgres, Slanger, Redis and also log files from the applications inside the container.
+
+The files are located at `/var/travis` on the platform machine. Please run `sudo tar -czvf travis-enterprise-data-backup.tar.gz /var/travis` to create compressed archive from this folder. After this has finished, copy this file off the machine to a secure location.
+
+## Builds don't get worked off
+
+### Symptoms
+
+In the Travis CI Web UI you see none of the builds getting worked off. They're either in no or the `queued` state. Cancelling and restarting them doesn't make any difference.
+
+### Strategies
+
+Below you will find different strategies to resolve the problem. They're meant to be followed in order. After you've completed the steps for a strategy please restart a build in the Travis CI Web UI to see if it gets picked up. If that's not happening, please advance to next strategy.
+
+#### Connection to RabbitMQ got lost
+
+We're using RabbitMQ to schedule builds for the worker machine(s). Sometimes it can happen that the worker machine(s) lose the connection to RabbitMQ and therefore don't run any new builds anymore. This is a known problem on our side and we're working on resolving this. To get everything back to normal, restarting the machines usually suffices. To do that, connect via `ssh` and run the following command:
```

Comment:
I think `it can happen` can just be `it happens` - the `sometimes` qualifies enough, I think?